buildenv/jenkins/JenkinsfileBase

```diff
@@ -207,7 +207,12 @@ def setupParallelEnv() {
 				// check for each lib. If lib does not exist, donwload it.
 				// If lib exists, SHA will be checked. Re-download if SHA does not match.
 				timeout(time: 20, unit: 'MINUTES') {
-					def customUrl = "https://ci.adoptium.net/job/test.getDependency/lastSuccessfulBuild/artifact/"
+					def jenkinsDomain = getJenkinsDomain()
+					if (jenkinsDomain.contains("hyc-runtimes")) {
+						jenkinsDomain = "openj9-jenkins.osuosl.org"
+					}
+					def customUrl = "https://${jenkinsDomain}/job/test.getDependency/lastSuccessfulBuild/artifact/"
+					echo "Custom URL: ${customUrl}"
```

Comment:
Please create a function for this. We should not duplicate the code.

Reply:
Created function to avoid duplicate code.
Grinder link:
https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/40359/

Reply:
Grinder link:
https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/40613/